src/components/Card/Card.js

```diff
@@ -22,11 +22,11 @@ const channelsList = channels =>
         href={url}
         target="_blank"
         rel="noopener noreferrer"
+        title={channel.type}
       >
         <div className="icon">
           <i className={`fa fa-${icon}`} />
         </div>
-        <div className="type">{channel.type}</div>
```

Comment:
I want to keep the labels for accessibility purposes

Reply:
We can, however remove the underline

Reply:
No problem, will revert it in next commit